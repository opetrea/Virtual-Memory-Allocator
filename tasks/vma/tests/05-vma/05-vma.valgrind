==91841== Memcheck, a memory error detector
==91841== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==91841== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==91841== Command: /home/student/Desktop/tema1_sda/sd-2023-homeworks/vma
==91841== Parent PID: 91840
==91841== 
==91841== 
==91841== HEAP SUMMARY:
==91841==     in use at exit: 0 bytes in 0 blocks
==91841==   total heap usage: 17 allocs, 17 frees, 8,592 bytes allocated
==91841== 
==91841== All heap blocks were freed -- no leaks are possible
==91841== 
==91841== For lists of detected and suppressed errors, rerun with: -s
==91841== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
