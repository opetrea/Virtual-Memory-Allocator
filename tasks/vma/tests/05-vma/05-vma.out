Total memory: 0xA bytes
Free memory: 0x7 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 3

Block 1 begin
Zone: 0x1 - 0x4
Miniblock 1:		0x1		-		0x2		| RW-
Miniblock 2:		0x2		-		0x3		| RW-
Miniblock 3:		0x3		-		0x4		| RW-
Block 1 end
